changelog.d/20240202_172501_i_atlas_for_all.md

```diff
@@ -0,0 +1 @@
+- [Improvement] Enable `atlas pull` on all Micro-frontends. (by @omarithawi)
```

Comment:
nit: should this be an improvement or feature? I understand we are changing the translation process but it is an entirely new process overall.

Reply:
I have no idea. I'll make it `Feature` :)